manifests/params.pp

```diff
@@ -20,6 +20,25 @@
   $package_name       = 'vault'
   $package_ensure     = 'installed'
 
+  $checksum_type       = 'sha256'
+  $checksums           = {
```

Comment:
This is going to be a big maintenance burden, every version bump is going to require me to lookup all these checksums.